pom.xml

```diff
@@ -17,7 +17,7 @@
     <jmock.version>2.5.1</jmock.version>
     <mockito-core.version>4.0.0</mockito-core.version>
     <pentaho-scheduler-plugin.version>11.0.0.0-SNAPSHOT</pentaho-scheduler-plugin.version>
-    <pentaho-generic-file-system.version>1.0.0</pentaho-generic-file-system.version>
+    <generic-file-system.version>1.0.0</generic-file-system.version>
```

Comment:
Maybe be more explicit that the intention is to use a particular version that is local to the plugin.  Maybe `plugin-local-generic-file-system.version` ? Or maybe adding a comment.
This would be helpful conveying the intention to someone that isn't familiar with the project.

Reply:
Addressed in [e48bc2d](https://github.com/pentaho/pentaho-scheduler-plugin/pull/336/commits/e48bc2df8354be8faaed387c01dc3f0f432c6b7c).